packages/core/src/test/awsService/appBuilder/serverlessLand/main.test.ts

```diff
@@ -0,0 +1,224 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import assert from 'assert'
+import * as sinon from 'sinon'
+import * as path from 'path'
+import { MetadataManager } from '../../../../awsService/appBuilder/serverlessLand/metadataManager'
+import * as main from '../../../../awsService/appBuilder/serverlessLand/main'
+import {
+    createNewServerlessLandProject,
+    openReadmeFile,
+    getProjectUri,
+    downloadPatternCode,
+} from '../../../../awsService/appBuilder/serverlessLand/main'
+import { fs } from '../../../../shared/fs/fs'
+import * as downloadPatterns from '../../../../shared/utilities/downloadPatterns'
+import { ExtContext } from '../../../../shared/extensions'
+import { workspaceUtils } from '../../../../shared'
+import * as downloadPattern from '../../../../shared/utilities/downloadPatterns'
+import * as wizardModule from '../../../../awsService/appBuilder/serverlessLand/wizard'
+
+describe('createNewServerlessLandProject', () => {
```

Comment:
nit: You can deduplicate code by moving all these `describe()` in to a nested `describe()` and then in the top level you create the sandbox once